Formula/a/anyquery.rb

```diff
@@ -0,0 +1,33 @@
+class Anyquery < Formula
+  desc "Query anything with SQL"
+  homepage "https://anyquery.dev"
+  url "https://github.com/julien040/anyquery/archive/refs/tags/0.4.0.tar.gz"
+  sha256 "bcadb61a2af9ae23f81ca55318d2b3f263cdd4012b4d136d16be3462e623c3e2"
+  license "AGPL-3.0-only"
+  head "https://github.com/julien040/anyquery.git", branch: "main"
+
+  depends_on "go" => :build
+
+  def install
+    tags = %w[
+      vtable
+      fts5
+      sqlite_json
+      sqlite_math_functions
+    ]
+    system "go", "build", "-tags=#{Array(tags).join(" ")}", *std_go_args(ldflags: "-s -w")
```

Comment:
this would look better
```suggestion
    tags = %w[
      vtable
      fts5
      sqlite_json
      sqlite_math_functions
    ].join(",")
    system "go", "build", "-tags", tags, *std_go_args(ldflags: "-s -w")
```

Reply:
agree